src/network/networkd-radv.c

```diff
@@ -19,6 +19,44 @@
 #include "string-table.h"
 #include "strv.h"
 
+void network_adjust_radv(Network *network) {
```

Comment:
what is this commit precisely doing? i.e. te diff is too hard to read, and your commit just says "move functions"? so are oyu saying this commit just reorders the functions?

Reply:
Right. I will update the commit message later.